scripts/restore

```diff
@@ -96,10 +96,12 @@ chown root: $final_path/wp-config.php
 
 ynh_restore_file --origin_path="/etc/php/$phpversion/fpm/pool.d/$app.conf"
 
+ynh_add_fpm_config --package="$extra_php_dependencies"
```

Comment:
```suggestion
ynh_add_fpm_config --usage=$fpm_usage --footprint=$fpm_footprint --package="$extra_php_dependencies"
```

You must retrieve  `fpm_usage` and `fpm_footprint` in the `LOAD SETTINGS` section